### PR TITLE
Add dash, plus templates for db and redis

### PR DESCRIPTION
**Pull request by mikowitz**
* Add Tiltfile for Dash
* Add templates and related configuration for db and redis deployments

### CODE REVIEW

File: support/templates/db.yaml.template [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+  labels:
+    type: local
+spec:
+  storageClassName: manual
+  capacity:
+    storage: 50Gi
+  accessModes:
+    - ReadWriteOnce
+  hostPath:
+    path: "/var/lib/ez-tilt/dash/postgresql/data"
```
Comment (djcp) [inline support/templates/db.yaml.template]: If we're genericizing this, this should be templatted.

Comment (mikowitz) [reply support/templates/db.yaml.template]: Whoops, thought I'd done a file-wide replace, but must've missed this one. Fixed now.

File: support/templates/db.yaml.template [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+      labels:
+        app: __APP_NAME__-db
+    spec:
+      volumes:
+        - name: __APP_NAME__-db-storage
+          persistentVolumeClaim:
+            claimName: __APP_NAME__-db-claim
+      containers:
+      - name: __APP_NAME__-db
+        image: ezcater/postgres:9.4.9-postgis-2.1
```
Comment (djcp) [inline support/templates/db.yaml.template]: This probably deserves to be templatted, I've had to set different values for different services.

File: support/templates/db.yaml.template [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+        ports:
+        - containerPort: 5432
+        volumeMounts:
+          - mountPath: "/var/lib/postgresql/data"
```
Comment (djcp) [inline support/templates/db.yaml.template]: Also, I'm not sure if different psql images will have different data directories or not, but we might need to templatize this too. Maybe we can pass in a `dict` to allow for setting defaults so we can only override if needed?

Comment (djcp) [inline support/templates/db.yaml.template]: Also, I'm not sure if different psql images will have different data directories or not, but we might need to templatize this too. Maybe we can pass in a `dict` to allow for setting defaults so we can only override if needed?

**Comment by djcp** [comment]
I love the extended templatting in this branch. AFAIC, it's good to go whenever you're happy with it.

File: support/templates/db.yaml.template (hunk 3 above)
[the remark this answers is not in the document]
Comment (djcp) [reply support/templates/db.yaml.template]: I'm OK with you not doing that in this branch, though - we can templatize the `mountPath` the first time we need to

[the remark this answers is not in the document]
Comment (djcp) [reply support/templates/db.yaml.template]: I'm OK with you not doing that in this branch, though - we can templatize the `mountPath` the first time we need to

**Pull request merged**